app/workers/sync-In.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+require_once __DIR__ . '/../worker.php';
+
+use Appwrite\Extend\Exception;
+use Utopia\App;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Database\DateTime;
+use Utopia\Queue;
+use Utopia\Queue\Message;
+
+if (App::getEnv('_APP_REGION', 'default') === 'default') {
+    throw new Exception(Exception::GENERAL_SERVER_ERROR);
+}
+
+global $register;
+global $dsn;
```

Comment:
Use dependency injection, even if need to copy-paste. We are working on a mechanism for share dependencies between http, realtime, workers, cli.